### ISSUE TITLE
timit_preprocess.py

### ISSUE THREAD

**Issue by duxiansheng** [zzw922cn/Automatic_Speech_Recognition#2]
I can't read timit, the problem is ValueError: File format 'NIST'... not understood. can you help me?

**Comment by zzw922cn** [comment]
You should convert the format of your audio first~

**Comment by brianlan** [comment]
@duxiansheng I use sndfile converter to convert NIST to WAV. Please refer to [this repo](https://github.com/erikd/libsndfile) and their [home page](http://www.mega-nerd.com/libsndfile/).

**Comment by zzw922cn** [comment]
@duxiansheng @brianlan nst2wav script has been updated just now, it's in the feature folder.

**Comment by duxiansheng** [comment]
I can covert it with Auducity,but there are a lot of work to do. And is there a way to covert all of .wav by code?

**Comment by zzw922cn** [comment]
@duxiansheng yes, you can see my new code in timit or libri or wsj, it can automatically process both wav and nist files.
